bridge/client/app/_views/ktb-service-view/ktb-service-view.component.html

```diff
@@ -8,9 +8,18 @@
           </dt-info-group-title>
         </dt-info-group>
       </div>
-      <div fxLayout="row" fxLayoutAlign="start center" *ngIf="project.getServices().length == 0" uitestid="keptn-service-view-noServices">
-        <dt-icon class="icon" name="information"></dt-icon>
-        <p>No service onboarded yet. Follow the instructions to <a [href]="'/manage/service/#onboard-a-service' | keptnUrl" target="_blank" rel="noopener noreferrer">onboard a service</a>.</p>
+      <div fxLayout="row" fxLayoutAlign="start" class="mt-2" *ngIf="project.getServices().length == 0" uitestid="keptn-service-view-noServices">
+        <dt-icon fxFlex="20px" class="icon" name="information"></dt-icon>
+        <p class="m-0 ml-2">
+          <ng-container *ngIf="!isQualityGatesOnly">
+            No service onboarded yet. Follow the instructions to
+            <a [href]="'/manage/service/#onboard-a-service' | keptnUrl" target="_blank" rel="noopener noreferrer">onboard a service</a>.
+          </ng-container>
+          <ng-container *ngIf="isQualityGatesOnly">
+            No service created yet.
```

Comment:
`Follow the instructions to` is missing here

Reply:
Done